src/config/config.js

```diff
@@ -4,6 +4,11 @@ const config = {
   dayLabels: [
     'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday',
   ],
+  postTable: {
+    linkPrefix: 'https://reddit.com/',
+  },
 };
 
-export default config;
+const timezoneDifference = new Date().getTimezoneOffset() * 60;
```

Comment:
Btw: I still don't think we need to use the timezone difference. You can double-check the times in reddit when you hover over the time in the post. It doesn't match your table. (First you need to fix the links in the table though)